.buildkite/kind-in-docker.yml

```diff
@@ -34,3 +34,24 @@
     - kubectl cluster-info
     - kubectl get nodes
     - kubectl get pods --all-namespaces
+
+    # Install Helm
+    - curl -Lo helm.tar.gz https://get.helm.sh/helm-v3.12.2-linux-amd64.tar.gz
+    - tar -zxvf helm.tar.gz
+    - mv linux-amd64/helm /usr/local/bin/helm
+    - helm repo add kuberay https://ray-project.github.io/kuberay-helm/
+    - helm repo update
+
+    # Install KubeRay operator
+    - pushd ray-operator
+    - IMG=kuberay/operator:nightly make docker-build
+    - kind load docker-image kuberay/operator:nightly
+    - popd
+
+    - pushd helm-chart/kuberay-operator
+    - helm install kuberay-operator --set image.repository=kuberay/operator --set image.tag=nightly .
```

Comment:
You can also consider using the `--wait` option with Helm CLI (https://helm.sh/docs/helm/helm_install/) because `kubectl wait` isn't a smart command. Relying solely on `kubectl wait` might introduce flakiness. A lot of flakiness in KubeRay CI was introduced by `kubectl wait`. You can refer #618 for some examples.

In this PR, what will happen if the CI pipeline executes `kubectl wait` after the `helm install` command but before the KubeRay operator Pod is created? I guess it will report error.

Reply:
Good point, let me switch to helm `--wait`.

Reply:
Done.